### PR TITLE
<InputArea /> - refactor story page

### PR DESCRIPTION
**Pull request by yardenez**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

File: src/InputArea/docs/index.story.js [hunk 1]
```diff
@@ -38,47 +54,38 @@ export default {
...
+      sourceUrl: `https://github.com/wix/wix-style-react/tree/master/src/${InputArea.displayName}/`,
+    }),
+
+    tabs([
+      tab({
+        title: 'Description',
+        sections: [
+          description({
+            title: 'Description',
+            text: 'General inputArea container',
```
Comment (yardenez) [inline src/InputArea/docs/index.story.js:71]: Should I change the description to another text?

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6008

File: src/InputArea/docs/examples.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+export const standard = `
```
Comment (EgoziE) [inline src/InputArea/docs/examples.js:1]: Standard
A simple use with a placeholder
```
<InputArea placeholder="This is a placeholder" resizable />
```

Letter counting
InputArea can indicate how many letters was typed, in order to show the counter you should set both `maxLength` and `hasCounter`. 
```
<InputArea value ="I typed exactly 37 letters out of 100" resizable maxLength={100} hasCounter />
```

Disabled and Read only
InputArea supports `disabled` and `readOnly` states
```
<Layout>
  <Cell>
    <InputArea placeholder="This is a placeholder" resizable disabled />
  </Cell>
  <Cell>
    <InputArea placeholder="This is a placeholder" resizable readOnly />
  </Cell>
</Layout>
```

With status
InputArea supports `error`, `warning`, and `loading` status
```
<Layout>
  <Cell>
    <InputArea placeholder="This is a placeholder" resizable status="error" />
  </Cell>
  <Cell>
    <InputArea placeholder="This is a placeholder" resizable status="warning" />
  </Cell>
  <Cell>
    <InputArea placeholder="This is a placeholder" resizable status="loading" />
  </Cell>
</Layout>
```

File: src/InputArea/docs/index.story.js (hunk 1 above)
Comment (EgoziE) [reply src/InputArea/docs/index.story.js:71]: change to:

A multi-line text input component

**Review by EgoziE — CHANGES_REQUESTED** [review]

**Review by EgoziE — APPROVED** [review]

**Pull request merged**